maven-plugin-plugin/src/main/java/org/apache/maven/plugin/plugin/PluginReport.java

```diff
@@ -316,13 +307,13 @@ private PluginDescriptor extractPluginDescriptor()
 
         try
         {
-            List<ComponentDependency> deps = GeneratorUtils.toComponentDependencies( project.getRuntimeDependencies() );
+            List<ComponentDependency> deps = GeneratorUtils.toComponentDependencies( project.getArtifacts() );
             pluginDescriptor.setDependencies( deps );
 
             PluginToolsRequest request = new DefaultPluginToolsRequest( project, pluginDescriptor );
             request.setEncoding( encoding );
             request.setSkipErrorNoDescriptorsFound( true );
-            request.setDependencies( dependencies );
+            request.setDependencies( new LinkedHashSet<>( project.getArtifacts() ) );
```

Comment:
Are we adding artifacts or dependencies?

Reply:
fixed

Reply:
There's nothing we can do here anymore. Could be a different ticket to fix this inconsistency.